### PR TITLE
added support for RANSACRegressor

### PR DESCRIPTION
**Pull request by StrikerRUS**

### CODE REVIEW

File: m2cgen/assemblers/linear.py [hunk 1]
```diff
@@ -22,6 +22,15 @@ def _build_ast(self):
         return ast.VectorVal(exprs)
 
 
+class RANSACModelAssembler(ModelAssembler):
```
Comment (StrikerRUS) [inline m2cgen/assemblers/linear.py]: Not sure, may it's better to place this class into a separate module, let say `meta.py`, as it has nothing in common with linear models implementation...

Comment (izeigerman) [reply m2cgen/assemblers/linear.py]: Perhaps in this particular case it makes sense to unwrap the estimator right inside of the `get_assembler_cls` function, and don't introduce any extra entities on top of it. 
My reasoning here is based on the fact that in this case no additional logic is introduced by RANSAC specifically, only unwrapping takes place.
What do you think?

**Comment by izeigerman** [comment]
Thanks @StrikerRUS ! Sorry about the delay with PRs, was traveling these past 2 weeks. Left one comment, otherwise good to go.

File: m2cgen/assemblers/linear.py (hunk 1 above)
Comment (StrikerRUS) [reply m2cgen/assemblers/linear.py]: I tried the approach you are talking about firstly 🙂  Unfortunately, it failed due to that we can get assembler for the underlying model in `get_assembler_cls()`, but we cannot replace `model` with `model.estimator_` there. So, we we'll try to assemble `RANSACRegressor` (still) instance with the appropriate assembler (`LinearModelAssembler` let say for example).
https://github.com/BayesWitnesses/m2cgen/blob/1553c4cd0fb2493720e70639d460927110e7fc90/m2cgen/exporters.py#L255-L256

Also, I think that `RANSACRegressor` is not the last meta estimator we might want to support. The first thing that comes to my mind is `GridSearchCV`. We can add the support for it right now simply by changing `base_model = self.model.estimator_` to `base_model = self.model.best_estimator_`. That's why I want to move the implementation to `meta.py` and make generalizable approach for meta estimators.

Comment (izeigerman) [reply m2cgen/assemblers/linear.py]: > but we cannot replace model with model.estimator_ there.
Ah, good point indeed. 
I think your reasoning makes a lot of sense. I also like the idea of moving it to `meta.py` module 👍

Comment (StrikerRUS) [reply m2cgen/assemblers/linear.py]: Thanks! Done in the latest commit.

**Pull request merged**